src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -471,11 +471,21 @@ private void BeforeAttribute()
                 return;
             }
 
+            // Minimized attribute
+            // Need to modify this code to account for space inbetween equals when
+            // https://github.com/aspnet/Razor/issues/123 is completed.
             if (!At(HtmlSymbolType.Equals))
             {
-                // Saw a space or newline after the name, so just skip this attribute and continue around the loop
-                Accept(whitespace);
-                Accept(name);
+                // Output anything prior to the attribute, in most cases this will be the tag name:
+                // |<input| checked />
```

Comment:
This is the parsing change that enables the `TagHelper` rewriter to understand minimized attributes. Attributes are always grouped with the prefixed whitespace before them, prior to this change you would get a single block which contained `<input checked` where now you get one for `<input` and another with ` checked`.

Therefore, this comment is saying "we're about to construct a minimized attribute block, other info does not belong inside of it, Output that now before we construct the block dedicated to the minimized attribute". I'll nuke the comment all together, we don't add them in other places.